x-pack/plugins/licensing/server/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PluginInitializerContext } from 'src/core/server';
+import { schema } from './schema';
+import { Plugin } from './plugin';
+
+export * from './types';
```

Comment:
I'd expect `LicenseType`, `LICENSE_TYPE` and other types users interact with to be exported as well

Reply:
Fixed in next PR.